src/main/java/com/github/onsdigital/babbage/pdf/PDFGenerator.java

```diff
@@ -47,19 +56,22 @@ public static Path generatePdf(String uri, String fileName, Map<String, String>
                 html = html.replace("&nbsp;", "&#160;");
             }
 
-            String outputFile = TEMP_DIRECTORY_PATH + "/" + fileName + ".pdf";
-            InputStream inputStream = new ByteArrayInputStream(html.getBytes());
-            createPDF(uri, inputStream, outputFile);
-
-            Path pdfFile = FileSystems.getDefault().getPath(TEMP_DIRECTORY_PATH).resolve(fileName + ".pdf");
-            if (!Files.exists(pdfFile)) {
-                throw new RuntimeException("Failed generating pdf, file not created");
+            String outputFile = TEMP_DIRECTORY_PATH + "/" + fileName + PDF_EXT;
+            try (
+                    InputStream inputStream = new ByteArrayInputStream(html.getBytes());
+                    FileSystem fs = FileSystems.getDefault();
+                    FileReader fileReader = new FileReader(outputFile);
```

Comment:
this is where my java knowledge falls down 😀  is it safe to create a FileReader against a file which doesn't exist yet?